docs/src/pages/careers/lead-designer.md

```diff
@@ -48,35 +48,36 @@ It's why we started MUI Base, Joy UI, and MUI Toolpad. Design is key to achievin
 
 Depending on the day, you'll:
 
-- Make the product experience more delightful, e.g. visually more appealing
-- Pay attention to all the smallest interface interactions so they yield an outstanding UX
-- Develop the brand
-- Refresh and expand the marketing pages, e.g. with a showcase & case study section
-- Design the next features across the different product
-- Shape the direction of the design kit products
-- **Solve organization issues**
-  - You will improve how the design responsibilities are covered.
-  - You will ensure the consistency of design and product direction throughout all the products and all customer-facing artifacts.
-- **Exercise team leadership**
-  - You will overseas the overall design quality
-  - You will manage designer's career
-  - You will hire and maintain a great team
+- Ensure high-quality UI and UX across teams, libraries, components, and product strategy
+- Collaborate with product management and engineering to ensure that design work is aligned with business goals and user needs
+- Create design concepts, prototypes, and high-fidelity mockups for projects on the X, Core, and Toolpad teams
+- Update and implement new components on the Core, X, and Toolpad teams
+- Develop and maintain MUI's design system and style guide
+- Oversee the design kits, ensuring they're aligned with engineering implementation
+
+As a team leader, you will:
+
+- Manage the design process, including user research, user flows, and user testing
+- Review design work and provide feedback to ensure overall design quality and consistency, with consideration to both UI and UX.
```

Comment:
```suggestion
- Review design work and provide feedback to ensure overall design quality and consistency, with consideration to both UI and UX
```

Just removing that final period to make it consistent with the other bullet points in this section.